package.json

```diff
@@ -208,6 +214,7 @@
     "reselect-tools": "^0.0.7",
     "ssri": "^8.0.1",
     "storybook-dark-mode": "^1.0.9",
+    "ts-jest": "^27.1.4",
```

Comment:
Included with "react-scripts"

Reply:
You won't be able to use it with the jest transform config unless it's specified as a dev-dep directly. But I've removed that from this PR since it's unrelated and removed this.